cmd/commands/connection/command.go

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package connection
+
+import (
+	"github.com/mysteriumnetwork/node/cmd/commands/cli/clio"
+	"github.com/mysteriumnetwork/node/config"
+	"github.com/mysteriumnetwork/node/config/urfavecli/clicontext"
+	"github.com/mysteriumnetwork/node/core/node"
+	tequilapi_client "github.com/mysteriumnetwork/node/tequilapi/client"
+
+	"github.com/urfave/cli/v2"
+)
+
+// CommandName is the name of this command
+const CommandName = "connection"
+
+var (
+	flagCountry = cli.StringFlag{
+		Name:  "country",
+		Usage: "Two letter (ISO 3166-1 alpha-2) country code to filter proposals.",
+	}
+
+	flagLocationType = cli.StringFlag{
+		Name:  "location-type",
+		Usage: "Node location types to filter by eg.'hosting', 'residential', 'mobile' etc.",
+	}
+)
+
+const (
+	serviceWireguard = "wireguard"
+
+	gbInBytes = 1 << (10 * 3)
```

Comment:
We have it already defined in the https://github.com/mysteriumnetwork/node/blob/master/datasize/bitsize.go